routing/http/client/client.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	_                 contentrouter.Client = &client{}
+	_                 contentrouter.Client = &Client{}
```

Comment:
Given https://specs.ipfs.tech/routing/http-routing-v1/ maybe call this `DelegatedRoutingClient` ?

I imagine someone integrating IPFS may have multiple "clients" (DNS, Kubo RPC, Pinning service, and now this), better to have meaningful names?

Reply:
@lidel hmm, that feels a bit long. I think the main issue here is also the package name. It';s just called `client`. If it were called something else, it would be much more explicit, like `delegatedrouting.Client`.